lib/msf/core/post/file.rb

```diff
@@ -286,11 +287,16 @@ def readable?(path)
         return cmd_exec("[System.IO.File]::OpenRead(\"#{path}\");if($?){echo\
           #{verification_token}}").include?(verification_token)
       end
+    elsif session.platform == 'windows'
+      result = cmd_exec("type #{path}").to_s
+      if result.include?("Access is denied") || result.include?("cannot find the file")
```

Comment:
Is this going to work when the language pack isn't English?

Reply:
Uhh good point, I don't think it will.

Reply:
I'm going to rewrite this code to use the new `icacls` and `whoami /group` approach that I mentioned above. Will mark this as a draft until I can get the code for this worked out.

Reply:
Reworked this in https://github.com/rapid7/metasploit-framework/pull/15987/commits/a204591f98006a89996bc105b6f15a5680bad099, closing this as its outdated.